database/buildSchema/34_b_verification.sql

```diff
@@ -0,0 +1,21 @@
+-- verification
+CREATE TABLE public.verification (
+    id serial PRIMARY KEY,
+    unique_id varchar UNIQUE NOT NULL,
+    application_id integer REFERENCES public.application (id) NOT NULL,
+    code varchar,
+    message varchar,
+    data jsonb,
+    created_time timestamptz DEFAULT NOW(),
+    expiry_time timestamptz,
```

Comment:
I would suggest `time_created` so it has the same pattern as other places that are using similar timestamps. And I'm still reading this but `expiry_time` maybe better as `expiry_date`?

Reply:
Now that I found that this is a duration and not a date, I suggest `duration_until_expired` and perhaps another one saying its expired?

Reply:
Okay, how about `expiry_hours`, so it's clear what the units are as well?

I considered making a field for showing whether it's expired or not, but I figured that the logic required to ensure it's always up to date and accurate would be more complicated than just deducing it every time you need to check it (ie. `if time_created + expiry_hours < current time...`). Can you think of a use case that would justify stating it explicitly?

Reply:
> Now that I found that this is a duration and not a date, I suggest `duration_until_expired` and perhaps another one saying its expired?

Hang on, I was mistaken --  the `expiry_time` field _is_ a date/timestamp -- the "expiry" field in the Action input is a duration. The expiry time is calculated when the record is created:
```js
 const expiryTime = expiry ? DateTime.now().plus({ hours: expiry }).toISO() 
```

So I'll call the fields `time_created` and `time_expiring`, okay? (the expiring one is a bit ambiguous because it could actually be "expiring" or "expired" depending on if the time has passed or not.)

Reply:
> (the expiring one is a bit ambiguous because it could actually be "expiring" or "expired" depending on if the time has passed or not.)

Which is why `expiry_time` is clearer, although I take your point about consistency with other cases. I think I'll go with `time_expired` -- at least the tense is consistent with others, even if it's strictly incorrect if that time hasn't passed yet.

Reply:
Righto -- done! 5b5eeb76

Reply:
Ok. Sorry I didn't read that before. So now `time_expired` is the actual time when it will expire. That's all good by me